common/persistence/visibility/defs.go

```diff
@@ -47,6 +48,91 @@ func DefaultAdvancedVisibilityWritingMode(advancedVisibilityConfigExist bool) st
 	return SecondaryVisibilityWritingModeOff
 }
 
+//nolint:revive
+func ResolveVisibilityPersistenceMaxReadQPS(
+	dc *dynamicconfig.Collection,
+	advancedVisibilityStoreConfigExists bool,
+) dynamicconfig.IntPropertyFn {
+	if dc.HasKey(dynamicconfig.VisibilityPersistenceMaxReadQPS) {
+		return dc.GetIntProperty(dynamicconfig.VisibilityPersistenceMaxReadQPS, 9000)
+	}
+	if advancedVisibilityStoreConfigExists {
+		return dc.GetIntProperty(dynamicconfig.AdvancedVisibilityPersistenceMaxReadQPS, 9000)
+	}
+	return dc.GetIntProperty(dynamicconfig.StandardVisibilityPersistenceMaxReadQPS, 9000)
+}
```

Comment:
This function (and the others) is to resolve the different existing configs into a single one in a backwards compatible way.

Reply:
Why `Resolve*`? It is wrapper over different gets. Just `Get*`?